tensorflow/core/util/mkl_util.h

```diff
@@ -2155,6 +2155,17 @@ inline bool IsConv1x1StrideNot1(memory::dims filter_dims,
           ((strides[0] != 1) || (strides[1] != 1)));
 }
 
+#ifdef ENABLE_MKLDNN_V1
+void execute_primitives(
+    std::vector<mkldnn::primitive>& primitives, std::shared_ptr<stream> stream,
+    std::vector<std::unordered_map<int, memory>>& net_args) {
+  DCHECK_EQ(primitives.size(), net_args.size();
```

Comment:
(This should be automatically fixed once you sync with master.)
```suggestion
  DCHECK_EQ(primitives.size(), net_args.size());
```

Reply:
I have updated and fix the clang-format error. Please let me know, if this fix looks ok.